downscale_ratio: 0.3


subscribe_topic: '/camera/color/image_raw'
publish_topic: '/camera/color/small_image'
